docs/ci/ci.md

```diff
@@ -0,0 +1,79 @@
+# Continuous Integration
+
+## Gitlab
+
+[Gitlab Runners](https://docs.gitlab.com/ee/ci/runners/README.html) run the code defined in your `.gitlab-ci.yml` project's file.
```

Comment:
I don't think we need this part, I assume people already aware of GitLab if they look for the integration solution